folium/features.py

```diff
@@ -241,8 +241,19 @@ def __init__(self, data, style_function=None):
             else:  # This is a filename
                 self.embed = False
                 self.data = data
+        elif data.__class__.__name__ in ['GeoDataFrame','GeoSeries']:
+            if hasattr(data, '__geo_interface__'):
+                # We have a GeoPandas 0.2 object
+                self.embed = True
+                self.data = json.loads(json.dumps(data.to_crs(epsg='4326').__geo_interface__))
```

Comment:
> No, otherwise, the Exception 'Unable to transform this object to a GeoJSON.' will be replaced by Unknown method 'to_crs' (or something like this).

I was thinking of a copy like:

`a = data.to_crs(epsg='4326')`

 and then

`a.__geo_interface__` or  `a.to_json()`

But that way is fine.